lib/MT/Log.pm

```diff
@@ -285,19 +283,19 @@ sub list_props {
                     text  => 'debug'
                 },
                 {   label => MT->translate('Security or error'),
-                    value => SECURITY() | ERROR(),
+                    value => join(',', (SECURITY(), ERROR())),
                     text  => 'sercurity_or_error',
                 },
                 {   label => MT->translate('Security/error/warning'),
-                    value => SECURITY() | ERROR() | WARNING(),
+                    value => join(',', (SECURITY(), ERROR(), WARNING())),
                     text  => 'security_or_error_or_warning',
                 },
                 {   label => MT->translate('Not debug'),
-                    value => SECURITY() | ERROR() | WARNING() | INFO(),
+                    value => join(',', (SECURITY(), ERROR(), WARNING(), INFO())),
```

Comment:
notice ?

Reply:
@charsbar Thanks! I fixed it https://github.com/movabletype/movabletype/pull/1689/commits/c7673b340685d158a69900ac802ee0ea938a147b.